website/src/theme/Blog/Components/Author/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
```

Comment:
what did you change in this file?

Reply:
Only one `className`, but I can also overwrite it via CSS.

Reply:
There as also few changes in module CSS file. Just removed the code file locally, but it looks like the style file is not used, when JS part is gone.

Reply:
Yes we need the actual component to import the CSS.

What could help reduce the swizzle code a bit is to wrap the component instead of ejecting it, and then doing `<WrapperComponent className={styles.myCustomClassName}/>`. That's a trick to know about, but it is not always the most convenient in all situations.